pokemongo_bot/cell_workers/handle_soft_ban.py

```diff
@@ -9,7 +9,10 @@
 
 
 class HandleSoftBan(BaseTask):
-    def work(self):
+    def work(self, *args, **kwargs):
```

Comment:
this worker runs in less than 50ms when `should_run()` returns false, this is probably unneeded